app/client/src/features/form/components/sections/Profile.js

```diff
@@ -34,8 +74,51 @@ function Profile() {
         label="Link"
         placeholder="mycoolportfolio.com/myname"
       />
+      {selectedTemplate === 10 && [
```

Comment:
nit: One last tiny thing, would you mind using React's [fragment syntax](https://reactjs.org/docs/fragments.html#short-syntax) here instead of an array?

Reply:
Done.